README.md

```diff
@@ -1,17 +1,28 @@
 # libecl [![Build Status](https://travis-ci.org/Statoil/libecl.svg?branch=master)](https://travis-ci.org/Statoil/libecl)
 
 
-## Coming from OPM?
+`libecl` is a package for reading and writing the result files from
+the Eclipse reservoir simulator. The file types covered are the
+restart, init, rft, summary and grid files. Both unified and
+non-unified and formatted and unformatted files are supported.
 
-`libecl` is a package for handling an ensemble of reservoir models, an
-important part of that is beeing able to read and write the files from
-standard reservoir applications; `libecl` has quite extensive support for
-reading and writing the result files from the ECLIPSE reservoir
-simulator. The capabilities to read and write ECLIPSE result files is
-used by the OPM simulator codes.
+libecl is mainly developed on linux and osx, in addition there is a
```

Comment:
I think it is nicer to read when libecl is emphasized in some manner, either using `libecl`as above or maybe something like _libecl_.

_Linux_ and _Windows_ are names and should be capitalized. Same goes for _OS X_, that actually goes for _macOS_ these days..